mock_emg_client.py

```
@@ -0,0 +1,292 @@
+    ("REST",  WINDOWS_PER_2SEC,  "최종 REST",       1.0, 1.0),
+]
+
+
+# ══════════════════════════════════════════════════════════════
+# WebSocket EMG 전송
+# ══════════════════════════════════════════════════════════════
+async def run(include_failsafe: bool):
+    print(f"  WebSocket 연결 중: {WS_URL}")
+
+    async with websockets.connect(WS_URL) as ws:
+        print(f"  연결 성공! EMG 데이터 전송 시작\n")
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)